package.json

```diff
@@ -9,11 +9,10 @@
     "dist"
   ],
   "scripts": {
-    "test": "jest && npm run test-types",
-    "test-types": "tsc --noEmit -p ./tests/tsconfig.tests.json",
+    "test": "jest",
     "lint": "eslint ./src --ext .ts",
     "format": "prettier --write 'src/**/*.ts?(x)' && npm run lint -- --fix",
-    "typecheck": "tsc --noEmit",
+    "typecheck": "tsc -b",
```

Comment:
Now, this typecheck command checks the entire repo, including the typecheck tests.

Reply:
Doesn't this create artifacts, though? Shouldn't it be `tsc -b --noEmit`? 

The whole intention of the `typecheck` script is to typecheck without generating artifacts.

Reply:
No, noEmit has been moved to the config file - this is so that users running `tsc` without using the command don't accidentally generate artifacts.